Libplanet/Net/Messages/Message.cs

```diff
@@ -51,9 +51,9 @@ internal enum MessageType : byte
             GetTxs = 0x08,
 
             /// <summary>
-            /// Message containing serialized block.
+            /// Message containing serialized blocks.
             /// </summary>
-            Block = 0x09,
+            Blocks = 0x09,
```

Comment:
At least in theory, it's a distinct message type which is incompatible with the previous `Block` message type, and we'd released multiple Libplanet versions used `Block` message type, IMHO we'd better bump the magic number here to `0x0a`.  @longfin Opinion?

Reply:
Basically, We should comply backward compatibility as you say. however, I think that there is a burden to leave the code that will not be used.

Considering libplanet's current user base, I guess this burden will be greater than the importance of backward compatibility.

Reply:
It don't think we should leave any code for removed message types, but what I suggest is only to use `0x0a` here instead of `0x09`.

Reply:
Oh, I see. it seems to be fine.